docs/en/source/callFlow.rst

```diff
@@ -211,18 +208,18 @@ History
 
 .. figure:: figures/flow-6.png
 
-The Figure is showing the operational flow of entity subscription in the Scorpio Broker system. Following are the marked steps interpretation:
+The Figure is showing the operational flow of history in the Scorpio Broker system. Following are the marked steps interpretation:
 
-1. An application calls the NGSI-LD compliant interface (exposed by service API gateway) to the history manager in the form of an HTTP POST request.
+1.An application calls the NGSI-LD compliant interface (exposed by service API gateway) to the history manager in the form of an HTTP POST request.
 
 2. The request enters in service API gateway.
 
- a. The service API gateway discovers the actual serving micro-service endpoints (where the incoming requests need to be forwarded) from discovery & registry service.
+ 2.1 The service API gateway discovers the actual serving micro-service endpoints (where the incoming requests need to be forwarded) from discovery & registry service.
 
- b.The service API gateway forwards the HTTP request to the History Manager micro-service.   
+ 2.2 The service API gateway forwards the HTTP request to the History Manager micro-service.
 
-3. The history manager now executes the EVA algorithm approach on the received payload and push payload attributes to Kafka topic “TEMPORALENTITY”.
+3. The history manager now executes the EVA algorithm approach on the received payload and push payload attributes to Kafka topic “TEMPORAL”.
 
 **Note**: History Manager must walk through each attribute at the root level of the object (except @id and @type). Inside each attribute, it must walk through each instance (array element). Then, it sends the current object to the Kafka topic TEMPORALENTITY.
 
-4. The history manager will keep listening to the “TEMPORALENTITY” topic and for any new entry and performs the relative operation in the database.
\ No newline at end of file
+4. The history manager will keep listening to the “TEMPORAL” topic and for any new entry and performs the relative operation in the database.
```

Comment:
[Minor] Please changes in flow diagram 6 same as above.

Reply:
Done